src/utilities/create-coin.ts

```diff
@@ -0,0 +1,18 @@
+interface CreateCoin {
+  (
+    coin: { id: string; symbol: string },
+    balance: number,
+    onWatchlist: boolean
+  ): object;
```

Comment:
`object` is not good using here

you'd better type this out for safety

Reply:
👍  done